### ISSUE TITLE
Closing stale issues

### ISSUE THREAD

**Issue by gizahNL** [CasparCG/server#1407]
### Description
Currently there are 302 issues (after this one: 303 issues) open. Some are already implemented/fixed, some are duplicates, others nobody cares about anymore.

### Solution suggestion
Go through the issues, close those that are fixed/implemented/duplicates, for others ask in the issue if the person still cares, and if not, close after say 2 weeks.

**Comment by Julusian** [comment]
The problem is that it requires someone to have the time to sit and read through them all, and also have a good understanding to figure out what should be closed. 
If you find any which should be, then post a comment on them stating why and I can handle it

**Comment by gizahNL** [comment]
Perhaps using something like: https://github.com/marketplace/stale would help as well?

**Comment by Julusian** [comment]
I hate that stale bot. I've had or found so many bug issues in other projects closed because they are 'stale' but are still very valid. Some of them because I gave up on replying 'not stale' after a few months and no interaction from a maintainer

**Comment by gizahNL** [comment]
Yeah, it only works if maintainers are able to interact in a timely manner with issues as well. I unfortunately don't have enough inside knowledge in CasparCG to be able to tell what can definitely be closed or not, otherwise I'd volunteer for it.

**Comment by dimitry-ishenko** [comment]
@Julusian @dotarmin

Could you guys please clarify current status of the project? Like who the current maintainers are, if SVT is still involved/interested in the project, roadmap, etc.

That would be really helpful. As @gizahNL pointed out, there are many open issues and PRs that just keep rotting. I can't tell if this project is dead/dying or what.

**Comment by Julusian** [comment]
As far as I'm aware is that @dotarmin and @5opr4ni have left SVT, no are no longer involved in the project on that side. I do not know who is now in charge of the project at SVT, I have not had any communication from them.
@ronag is still doing some CasparCG things, but mostly focused on a fork used by nxtedition, with some of the changes making their way back here.

I know of many places that are using CasparCG, so I don't expect it to die anytime soon, but we are definitely in a quiet patch in development of the project. There isn't a roadmap of any kind.

I am still very much around, doing CasparCG things on behalf of both NRK and SuperFly. I don't have much time dedicated to CasparCG, it is mostly allocated as needed if NRK needs something or if something particularly motivates me.

The recent 2.4.0 beta1 was primarily funded by NRK, as they have made a push to update their fork from 2.1 to 2.3 which was necessary to get a newer version of CEF. And any miscellaneous small features and fixes recently have been funded by SuperFly. 

So at this point, the project is ticking along. There are quite a few open bugs which I hope to gradually look at, and I will pick off some small features if they appeal to me, but don't expect many large new features unless they are contributed or funded by someone.

I admit that many of the open PRs are a bit stuck and stale, and for most the authors appear to have moved on. I am trying to keep on top of new ones, but I don't know how to progress on the older ones. I don't feel I can justify closing many of them, and am not confident enough in them to merge them, so they are stuck waiting
